internals/secrethub/generate.go

```diff
@@ -91,7 +91,7 @@ func (cmd *GenerateSecretCommand) before() error {
 		charset = charset.Add(randchar.Symbols)
 	}
 
-	var includedCharsets []randchar.Charset
+	includedCharsets := make([]randchar.Charset, 0)
```

Comment:
We already know what the length of the includedCharsets is going to be, right? So, we can preallocate with the length of the includes:
```Go
includedCharsets := make([]ranchar.Charset, 0, len(cmd.includes))
```

Same goes for the options; we can preallocate the underlying array with a length of the number of min flags.